packages/patternfly-4/content/design-guidelines/usage-and-behavior/lists-and-tables/design.md

```diff
@@ -2,145 +2,142 @@
+7. **Pagination footer:** when present, provides navigation to additional pages
 
 #### When to use
-Use a Data Table when the information you want to display is fits into a structured, tabular format, i.e. has distinct rows and columns.
+Use a standard data table when the information you want to display fits into a structured, tabular format (i.e. has distinct rows and columns).
 
 #### When not to use
-The Data Table is not recommended for less structured or variably structured data that cannot be easily organized into columns. In these situations a Data Table should be used.
+Do not use for less structured or variably structured data that cannot be easily organized into columns. In these situations, use a [data list](#data-lists).
 
 #### How to use
-Consider the structure of the data you want to display and organize that information into columns. Columns will typically have column headers. Every row within a table must have a consistent format, and if the table row includes actions, they should always be placed in the rightmost column(s).
+Consider the structure of the data you want to display and organize that information into columns. Columns will typically have column headers. Every row within a table must have a consistent format. If the table row includes actions, they should always be placed in the rightmost column(s).
 
-### Compact Data Table
+### Compact data table
 
 ![compact data table](img/compact-data-table.png)
 
-When to use
-Use a Compact Data Table when you want to show as much data per page as possible and readability is a secondary concern.
+#### When to use
+Use a compact data table when you want to show as much data per page as possible and readability is a secondary concern.
 
 
-### Expandable Data Table
+### Expandable data table
 
 ![expandable data table](img/expandable-data-table.png)
 
-1. **Expansion:** expands the row.
-2. **Expansion panel:** contains details associated with a row.
+1. **Expansion:** expands the row
+2. **Expansion panel:** contains details associated with a row
 
-When to use
+#### When to use
 Use when you have more information than will comfortably fit inside a row or you want to provide a way for advanced users to access information that is not applicable to all users.
 
-### Compound-Expandable Data Table
+### Compound-expandable data table
 
 ![compound expandable data table](img/compound-expandable-data-table.png)
 
-1. **Expandable cell:** The user can click on any expandable cell to reveal more detail about an item. If the expansion for an item is already open, clicking on a different cell will close the current item and open a new one.
+1. **Expandable cell:** a cell that can be clicked to reveal more detail about an item. If the expansion for an item is already open, clicking on a different cell will close the current item and open a new one.
 2. **Expansion panel:** contains details associated with an expandable item.
 
-#### When to use
+**When to use**
```

Comment:
Should this be a level 4 heading like the others?

Reply:
Good catch! Updated. Thanks!